src/sensitivity.jl

```diff
@@ -39,10 +39,12 @@ to add to the function signature can be specified in `kwargs`, which must be a `
 macro explicit_intercepts(
     f::SymOrExpr,
     type_tuple::Expr,
-    is_node::Expr=:([true for _ in $(get_types(get_body(type_tuple)))]),
+    is_node::Expr=Expr(:vect, (true for _ in get_types(get_body(type_tuple)))...),
```

Comment:
There exists a function `trues(dims)`

```suggestion
    is_node::Expr=Expr(:vect, trues(length(get_types(get_body(type_tuple))))...),
```

Though that is nor much prettier, maybe abstract it into a function that works out the length?

Reply:
Yours allocates an extra array.

Reply:
Yes.

```
julia> @benchmark Expr(:vect, trues(16)...)
BenchmarkTools.Trial:
  memory estimate:  1.17 KiB
  allocs estimate:  23
  --------------
  minimum time:     1.998 μs (0.00% GC)
  median time:      2.229 μs (0.00% GC)
  mean time:        2.775 μs (17.86% GC)
  maximum time:     3.568 ms (99.89% GC)
  --------------
  samples:          10000
  evals/sample:     9

julia> @benchmark Expr(:vect, (true for _ in 1:16)...)
BenchmarkTools.Trial:
  memory estimate:  1.08 KiB
  allocs estimate:  22
  --------------
  minimum time:     1.926 μs (0.00% GC)
  median time:      2.137 μs (0.00% GC)
  mean time:        2.645 μs (17.66% GC)
  maximum time:     3.170 ms (99.89% GC)
  --------------
  samples:          10000
  evals/sample:     10
```